advocacy_docs/pg_extensions/advanced_storage_pack/using.mdx

```diff
@@ -61,7 +61,7 @@ INSERT INTO iot (thermostat_id, recordtime) VALUES (5678, 03:44);
 When you select the data from the IoT table, you can see from the ctid location that the data with the same `thermostat_id` was clustered together:
 
 ```sql
-SELECT ctid, thermostat_id, recordtime FROM iot;
+SELECT ctid, thermostat_id, recordtime FROM iot ORDER BY CTID;
```

Comment:
The example output below needs to  actually be ordered by the CTID column, but don't just change the ctid column, actually move the full rows up and down until they are ordered by that column.  (This is because, if you don't move the full rows, then rows where thermostat_id = 456 won't show next to each other.)